src/frontend/css/pieces.css

```diff
@@ -0,0 +1,111 @@
+/* alpha */
+.alpha piece.pawn.white {background-image:url('data:image/svg+xml;base64,PHN2ZyB2aWV3Qm94PSIwIDAgMjA0OCAyMDQ4IiB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciPjxwYXRoIGQ9Ik0gNzM0LjA0OTM4LDk4MC44MDgxNiA5MzAuMDkwNzksNzg3Ljk1NTIzIGMgMCwwIC0xODkuMTMxNzMsLTgxLjU4NzI0IC03OS40MDE3NCwtMjg3Ljg1MDg3IDc5LjMzNjc1LC0xNDkuMTMyMjkgMzAyLjk5NTU1LC0xMTQuMzI2ODEgMzYxLjE4ODg1LDUwLjIwNDkxIDYzLjI1MDEsMTc4LjgyOTI2IC0xMTIuODE2NSwyNDAuMDU2NDUgLTExMi44MTY1LDI0MC4wNTY0NSBMIDEzMjQuOTQzLDk4Ni43ODk2MyBaIE0gNDk4LjY5MzE1LDE3NzkuODA1NCBjIDAsMCAtNy44NTI1MywtMTA2Ljg3MzMgNTAuODAxODUsLTE1NC4yNjA4IDE5NS40ODc1NiwtMTczLjAxNTggMzM3LjI4NjY1LC0zODUuMzcwNiAzNzAuODg3MjYsLTU5OC45MTY1IGwgMjA5LjIzNDE0LDEuOTkzNSBjIDMzLjM0ODQsMjA2LjU1MzcgMTgyLjUwMDYsNDQ3LjAwNDEgMzIxLjAzNzIsNTYxLjc0NzEgMTAxLjczNiw1OS4wNTI0IDk4Ljk4NDIsMTk4LjM5MzcgOTguOTg0MiwxOTguMzkzNyB6IiBmaWxsPSIjZjlmOWY5Ii8+PHBhdGggZD0iTTUyMCAxNzY5aDEwMDhxOC05Ny0xMzItMTgyLTEzMi0xMDEtMTk2LjUtMjM5LjVUMTEyMCAxMDM5SDkyOHEtMTUgMTcwLTc5LjUgMzA4LjVUNjUyIDE1ODdxLTE0MSA4NS0xMzIgMTgyem01MDQgNzRINDQ2di03NHEtNC04MCA0MS41LTEzN1Q2MTMgMTUyNHExMTctOTEgMTcxLjUtMjE3LjVUODYzIDEwMzlINTc2bDI4NC0yMzlxLTg2LTc0LTg2LTE4OCAwLTEwMyA3My0xNzd0MTc3LTc0cTEwMyAwIDE3Ni41IDc0dDczLjUgMTc3cTAgMTE0LTg2IDE4OGwyODQgMjM5aC0yODdxMjMgMTQxIDc4IDI2Ny41dDE3MiAyMTcuNXE3OSA1MSAxMjQuNSAxMDh0NDIuNSAxMzd2NzR6TTc1NiA5NzRoNTM2bC0yMjUtMTkxcTEzNC0zMSAxMzQtMTcxIDAtNzYtNTIuNS0xMjYuNVQxMDI0IDQzNXEtNzMgMC0xMjUgNTAuNVQ4NDcgNjEycTAgMTQwIDEzNCAxNzF6IiBmaWxsPSIjMTAxMDEwIi8+PC9zdmc+')}
```

Comment:
This whole file seems to be just redefining the CSS from the individual files in `../theme/piece`

Reply:
Yes. `../theme/piece` at this point is just holding all the possible pieces for convenience. This file is making this smaller collection of piece themes available to the chessground by appending a new class name.

Reply:
Gotcha, understood